packages/astro/src/vite-plugin-adapter-config/index.ts

```diff
@@ -0,0 +1,41 @@
+import type { Plugin as VitePlugin } from 'vite';
+import type { AstroSettings } from '../types/astro.js';
+
+const VIRTUAL_CLIENT_ID = 'astro:adapter-config/client';
```

Comment:
The presence of `/client` implies a possible `/server`. However, it seems that's not the case, so maybe we should just remove it or call it `/runtime`